src/ape/api/networks.py

```diff
@@ -391,11 +395,25 @@ def required_confirmations(self) -> int:
         Returns:
             int
         """
-        try:
-            return self.config[self.name]["required_confirmations"]
-        except KeyError:
-            # Is likely a 'development' network.
-            return 0
+        return self._network_config.get("required_confirmations", 0)  # type: ignore
+
+    @property
+    def approximate_block_time(self) -> int:
+        """
+        The approximate amount of time it takes for a new block to get mined to the chain.
+        Configure in your ``ape-config.yaml`` file.
+
+        Config example::
+
+            ethereum:
+              mainnet:
+                approximxate_block_time: 15
```

Comment:
would `block_time` be just as right? these things are parameters that different chains target

Reply:
Okay, yeah if that is something generally understood.